Source/WebCore/loader/ResourceMonitor.cpp

```diff
@@ -159,6 +168,10 @@ void ResourceMonitor::updateNetworkUsageThreshold(size_t threshold)
     if (m_networkUsageThreshold == threshold)
         return;
 
+    RefPtr frame = m_frame.get();
+    if (!frame)
+        return;
+
     RESOURCEMONITOR_RELEASE_LOG("Update network usage threshold: threshold=%zu", threshold);
```

Comment:
The fact that `RESOURCEMONITOR_RELEASE_LOG` uses `m_frame` and that the call site has to know to null check it is not obvious at all. This is bad design that will cause more bugs in the future. I think we need to update `RESOURCEMONITOR_RELEASE_LOG` to null check m_frame internally.